src/components/UploadEditor.vue

```diff
@@ -57,7 +57,7 @@
 				Dismiss
 			</button>
 			<button class="primary" @click="handleUpload">
-				Upload
+				Send
```

Comment:
```suggestion
				{{ t('spreed', 'Send') }}
```

🙈 I hope "Send" is already translated